e2e/utils/test-utils.ts

```diff
@@ -1,9 +1,77 @@
-import {Page} from 'puppeteer';
+// import {performance} from "perf_hooks";
+import {JSHandle, Page} from 'puppeteer';
 import Checkbox from 'app/element/checkbox';
 import RadioButton from 'app/element/radiobutton';
 import TextOptions from 'app/element/text-options';
 import Textarea from 'app/element/textarea';
 import Textbox from 'app/element/textbox';
+import GoogleLoginPage from 'app/page/google-login';
+import HomePage from 'app/page/home-page';
+
+export async function signIn(page: Page) {
```

Comment:
`signIn` method moved here from`app-utils.ts`

Reply:
Utils classes in general tend to become like the junk drawer in your kitchen. It tends not to be a clear abstraction that's easy to maintain.